shell/platform/android/io/flutter/embedding/android/FlutterImageView.java

```diff
@@ -188,26 +169,23 @@ public boolean acquireLatestImage() {
     if (!isAttachedToFlutterRenderer) {
       return false;
     }
-    // There's no guarantee that the image will be closed before the next call to
-    // `acquireLatestImage()`. For example, the device may not produce new frames if
-    // it's in sleep mode, so the calls to `invalidate()` will be queued up
-    // until the device produces a new frame.
+    // 1. `acquireLatestImage()` may return null if no new image is available.
     //
-    // While the engine will also stop producing frames, there is a race condition.
+    // 2. There's no guarantee that the `onDraw()` called after `invalidate()`.
+    // For example, the device may not produce new frames if
+    // it's in sleep mode or some special Android devices so the calls to `invalidate()` will be
+    // queued up
+    // until the device produces a new frame.
```

Comment:
would you mind trying to justify the comments, so the line width is a bit more even?

Reply:
@blasten OK, I will justify the comments.

Reply:
@blasten Done~